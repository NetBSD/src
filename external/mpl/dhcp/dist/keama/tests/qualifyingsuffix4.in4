# ddns-domainname (aka qualifying-suffix)

ddns-domainname ".biz";
ddns-updates true;

# embedded
class "foo" {
    ddns-domainname ".bar";
}
