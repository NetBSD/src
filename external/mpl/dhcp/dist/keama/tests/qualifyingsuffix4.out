{
  # ddns-domainname (aka qualifying-suffix)
  "Dhcp4": {
    "dhcp-ddns": {
      "qualifying-suffix": ".biz",
      "enable-updates": true
    },
    "client-classes": [
      # embedded
      {
        "name": "foo"
//      /// Only global qualifying-suffix is supported
//      "qualifying-suffix": ".bar"
      }
    ]
  }
}
